### PR TITLE
Verify with problems from Library Checker 

### PR DESCRIPTION
**Pull request by qryxip**
https://github.com/rust-lang-ja/ac-library-rs/pull/45#issuecomment-691448964

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -129,3 +129,47 @@ jobs:
...
+
+      - name: Verify
+        run: |
+          NAMES=(
+            convolution_mod
+            static_range_sum
+            sum_of_floor_of_linear
+            unionfind
+          )
+          for name in "${NAMES[@]}"; do
```
Comment (koba-e964) [inline .github/workflows/ci.yml:170]: We might want to first login to https://judge.yosupo.jp. Whose account are we using? Or are we going to create another account dedicated for that?

**Review by koba-e964 — COMMENTED** [review]
We might need permission to use https://judge.yosupo.jp/ for automated judging. Do we have permission already?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (qryxip) [reply .github/workflows/ci.yml:170]: Credentials are not required. I've just executed

```console
❯ oj d https://judge.yosupo.jp/problem/unionfind -ad /tmp/unionfind
```

without any credentials and it worked.

**Comment by qryxip** [comment]
> We might need permission

About server load or something? That might be true.

**Comment by qryxip** [comment]
No, we do not access to the real Library Checker. we just `git clone` [yosupo06/library-checker-problems](https://github.com/yosupo06/library-checker-problems) and build test cases on our side.

**Review by koba-e964 — APPROVED** [review]

**Pull request merged**